packages/platforms/browser/lib/index.ts

```diff
@@ -1,4 +1,4 @@
-export { DefaultRoutingProvider, default, onSettle } from './browser'
+export { DefaultRoutingProvider, default, onSettle, ReactRouterRoutingProvider } from './browser'
 export { type BrowserConfiguration } from './config'
-export { type RouteResolver, type RoutingProvider } from './routing-provider'
+export { type RouteResolver, type RoutingProvider, type StartRouteChangeCallback } from './routing-provider'
```

Comment:
this type is referred to in the docs but wasn't being exported https://docs.bugsnag.com/performance/integration-guides/web/customizing-routes/